swap_meet/clothing.py

```diff
@@ -1,2 +1,8 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, category="Clothing", condition=0.0, age=None):
+        super().__init__(category, condition, age)
```

Comment:
This setup makes it possible to pass in a value for category that is not "Clothing", which could result in an instance of Clothing that has some other category.
```suggestion
    def __init__(self, condition=0.0, age=None):
        super().__init__("Clothing", condition, age)
```